src/Blocks/Quote/index.js

```diff
@@ -3,6 +3,7 @@ import QuoteEdit from './Edit';
 import QuoteView from './View';
 import QuoteVariations from './variations';
 
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
@ana-oprea We should avoid as much as possible `eslint-disable` comments to fix eslint issues

Reply:
@avoinea I commited the fix